tests/test_VariableFilter.py

```diff
@@ -0,0 +1,53 @@
+from blocks.bricks import Bias, Linear, Sigmoid
+from blocks.filter import VariableFilter
+from blocks.graph import ComputationGraph
+from blocks.roles import BIASES, FILTERS, PARAMETER
+
+from theano import tensor
+
+
+def test_VariableFilter():
```

Comment:
The naming convention would be `test_variable_filter`, CamelCase is only for classes.